### PR TITLE
Adjust awg to plunger code

### PR DESCRIPTION
**Pull request by fvanriggelen**
- added another example data file
- adjusted the awg_to_plunger code, both measurement and analysis function, for the latter adding the option of the 'click' method, adding documentation
- adjusting the example

### CODE REVIEW

File: qtt/algorithms/awg_to_plunger.py [hunk 1]
```diff
@@ -34,52 +37,83 @@ def get_dataset(ds):
...
+    if fig is not None:
+        plt.figure(fig)
+    pts0, pts1 = plt.ginput(2)
+    slope = (pts1[1]-pts0[1])/(pts1[0]-pts0[0])
+    offset = (pts0[1] - slope*pts0[0])
+
+    return offset, slope
+
+#%% Main functions
+def measure_awg_to_plunger(station, gate, minstrument, srange=30, step = 0.5):
```
Comment (eendebakpt) [inline qtt/algorithms/awg_to_plunger.py]: Rename `srange` to `scanrange` which is more descriptive

File: qtt/algorithms/awg_to_plunger.py [hunk 2]
```diff
@@ -34,52 +37,83 @@ def get_dataset(ds):
         ds = qcodes.load_data(ds)
     return ds
 
-#%% Main functions
 
-def measure_awg_to_plunger(station, gate, minstrument, sweeprange=100, method='hough'):
-    """ Perform measurement for awg to plunger using 2D fast scan check?
+def click_line(fig=None):
+    """Defines a line throught two points. The points are choosen by clicking on a position in a plot, two times.
```
Comment (eendebakpt) [inline qtt/algorithms/awg_to_plunger.py]: Defines -> Define
throught -> through

File: qtt/algorithms/awg_to_plunger.py [hunk 3]
```diff
@@ -34,52 +37,83 @@ def get_dataset(ds):
...
     
     Args:
-        station (str): name of station
-        gate (int): plunger gate number
-        minstrument: 
-        sweeprange (int): sweeprange in mV?
-        method: Not used
-            
+        fig (int or None): number of figure to plot the points in, when None it will plot a figure
```
Comment (eendebakpt) [inline qtt/algorithms/awg_to_plunger.py]: plot a figure -> use the current figure

File: qtt/algorithms/awg_to_plunger.py [hunk 4]
```diff
@@ -34,52 +37,83 @@ def get_dataset(ds):
...
+    
     Returns:
-        result: (qcodes.DataSet) 
+        offset (float): offset of the line
+        slope (float): slope of the line
+    """
+    if fig is not None:
+        plt.figure(fig)
+    pts0, pts1 = plt.ginput(2)
+    slope = (pts1[1]-pts0[1])/(pts1[0]-pts0[0])
```
Comment (eendebakpt) [inline qtt/algorithms/awg_to_plunger.py]: This will crash if `pts1[0]==pts0[0]`

File: qtt/algorithms/awg_to_plunger.py [hunk 5]
```diff
@@ -88,17 +122,31 @@ def analyse_awg_to_plunger(result, method='hough', fig=None, verbose=1):
...
-        lines = cv2.HoughLines(edges, 1, np.pi/180, int(gray.shape[0]*.8))
+        imx = imx.astype(np.float64) * \
+        (100. / np.percentile(imx, 99))  # scale image
+        #to here
+        
+        gray = qtt.pgeometry.scaleImage(imx)
+
+        edges = cv2.Canny(gray, 50, 150, apertureSize=3)
 
+        #lines = cv2.HoughLines(edges, 1, np.pi/180, int(gray.shape[0]*.8))
```
Comment (eendebakpt) [inline qtt/algorithms/awg_to_plunger.py]: Remove commented out code

File: qtt/algorithms/awg_to_plunger.py [hunk 6]
```diff
@@ -88,17 +122,31 @@ def analyse_awg_to_plunger(result, method='hough', fig=None, verbose=1):
         ims, r = qtt.algorithms.images.straightenImage(
             im, imextent, mvx=istep, mvy=None)
         H = r[4]
+        
+        #added code from here
+        diffvals = {'dx': 0, 'dy': 1, 'dxy': 2, 'xmy': 'xmy', 'g': 'g'}
```
Comment (eendebakpt) [inline qtt/algorithms/awg_to_plunger.py]: Remove `diffvals`, just pass 'dx' to `cleanSensingImage`

**Comment by peendebak** [comment]
@fvanriggelen Some of the comments are not addressed yet.

**Pull request merged**